iina/Preference.swift

```diff
@@ -923,4 +927,38 @@ struct Preference {
     return T.init(key: key) ?? T.defaultValue
   }
 
+  /**
+   Older versions of IINA converted mpv color data into NSObject binary using the now-deprecated `NSUnarchiver` class.
```

Comment:
The documentation convention is to start with a very short description of the function, followed by a blank line and then go into further discussion about the details. You can see that in Xcode if you bring up the quick help for this function. You will see that Xcode puts this entire paragraph at the top and the other paragraphs later on under a `Discussion` header.